extension/CMakeLists.txt

```diff
@@ -1,3 +1,16 @@
+# TODO(Ziyi): This is a temporary workaround otherwise we have to explicitly export symbols in extensions.
```

Comment:
Can you open an issue on this? Eventually we should get rid of this workaround right?